ansible/roles/daliuge_runtime/tasks/daliuge.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Update and/or install pip packages
+  command: pip install -U pip setuptools
```

Comment:
Consider http://docs.ansible.com/ansible/latest/pip_module.html - this will get you a check phase you don't get with the command.